converter/src/test/java/gov/cms/qpp/conversion/decode/QualityMeasureScopedTest.java

```diff
@@ -36,7 +36,7 @@ void internalDecodeValidMeasure137V5Ipop() throws IOException, XmlException {
 	void internalDecodeValidMeasure137V5Denom() throws IOException, XmlException {
 		Node result = scopedConversion(QrdaScope.MEASURE_REFERENCE_RESULTS_CMS_V2, location);
 		long denom = pluckDescendants(result, TemplateId.MEASURE_REFERENCE_RESULTS_CMS_V2, TemplateId.MEASURE_DATA_CMS_V2)
-				.filter(node -> node.getValue(MeasureDataDecoder.MEASURE_TYPE).equals(SubPopulations.DENOM))
+				.filter(node -> node.getValue(MeasureDataDecoder.MEASURE_TYPE).equals(SubPopulationLabel.DENOM.name()))
```

Comment:
Perhaps flip these around like the IPOP one and use `hasAlias` for future proofing's sake.

Reply:
Disagree. YAGNI until proven otherwise